mkosi/qemu.py

```diff
@@ -1563,13 +1564,19 @@ def add_virtiofs_mount(
                 ),
             ),
         ) as proc:
+            # Immediately suspend the child process so we can optionally register the VM with
+            # systemd-machined before continuing.
+            proc.send_signal(signal.SIGSTOP)
```

Comment:
Isn't systemctl freeze/thaw better, so that the whole scope is frozen at the same time?

Reply:
I don't think there's much point in freezing the scope when we know there's only one process in it. Even if there were more, the qemu one is the only one writing to stdout so it should be fine to just freeze that one.

Reply:
There might be more in the future - but also it's a good chance to dogfood these APIs too

Reply:
I had to rework this and now we don't send SIGSTOP from mkosi itself anymore, so there's no opportunity to use systemctl freeze anymore here.